src/vs/base/common/product.ts

```diff
@@ -188,6 +188,7 @@ export interface IProductConfiguration {
 
 	readonly commonlyUsedSettings?: string[];
 	readonly aiGeneratedWorkspaceTrust?: IAiGeneratedWorkspaceTrust;
+	readonly entitlement?: IEntitlement;
```

Comment:
only a single entitlement?

Reply:
For now yes. I thought about enabling this for multiple entitlements but those APIs could differ and not have the same response shape. I'll enable this for now and if we do decide to take this route, we should look towards standardizing it.